src/test/validatorTest.ts

```diff
@@ -645,14 +645,15 @@ describe('validator', () => {
             expect(result['errors']['crit']).to.have.lengthOf(0);
         });
 
-        it('1 unquouted template format version should return an object with validTemplate = true, no crit errors, 1 warn error', () => {
-            const input = 'testData/valid/yaml/valid_unquoted_template_version.yaml';
-            let result = validator.validateFile(input);
-            expect(result).to.have.deep.property('templateValid', true);
-            expect(result['errors']['crit']).to.have.lengthOf(0);
-            expect(result['errors']['warn']).to.have.lengthOf(1);
-            expect(result['errors']['warn'][0]['message']).to.contain('AWSTemplateFormatVersion is recommended to be of type string');
-        });
+        // TODO: is this really needed?
```

Comment:
I can't remember, prehaps a git blame will shed more light on this test

Reply:
I've just tried doing just that but I haven't been able to come up with some definitive answer, this either means that my git foo isn't strong enough (highly likely) or that perhaps it was added in some mass refactoring ... in essence, being it's just a warning, I'm not really sure that this is critical nor do I know of an easy fix for this so I leave that up to you ...

Reply:
Lets remove this